code/modules/clothing/spacesuits/spacesuits.dm

```diff
@@ -32,6 +32,8 @@
 
 	var/obj/machinery/camera/camera
 	var/tinted = null	//Set to non-null for toggleable tint helmets
+	origin_tech = "{'materials':1}"
```

Comment:
Lower-tech than intended?

Reply:
Tell me what to put, I thought materials 1 made sense.

Reply:
It just feels weird that the spaceworthy helmet is lower tech than the physical-protection-only one, that's all